### DISCUSSION TITLE
Migration from WordPress to Wiki.js

### DISCUSSION THREAD

**Discussion by jsenergy1** [requarks/wiki#5224] [Help / Questions]
I'm working on getting a wiki that was built out in WordPress 5.9 migrated to the latest version of wiki.js. Is this possible? I could not find documentation or any articles detailing a migration path. This will be on a Linux RedHat host. Thanks in advance.

**Comment by NGPixel** [comment]
Look for options / plugins allowing you to export your pages to either markdown or HTML. You can then import these files using the Local Disk Storage module or via git.
